### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by jburel**
remove ice35

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -102,17 +102,6 @@
     </dependencies>
   </dependencyManagement>
   <profiles>
-    <profile>
-      <id>ice35</id>
-      <activation>
-        <property>
```
Comment (sbesson) [inline pom.xml]: Does this mean that by default no profile is activated at the moment? Should we activate `ice36` by default?

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
   <groupId>ome</groupId>
   <artifactId>pom-omero-client</artifactId>
   <packaging>pom</packaging>
-  <version>5.4.9</version>
+  <version>5.5.0-SNAPSHOT</version>
```
Comment (sbesson) [inline pom.xml]: Worth bumping to `5.5.0-m1` to start consuming it downstream. Or do you want to test things with `-SNAPSHOT` first?

Comment (mtbc) [reply pom.xml]: I could try with downloader if I knew how to set the ome.maven repository to fetch snapshots.

Comment (sbesson) [reply pom.xml]: Assuming `omero-downloader` consumes https://artifacts.openmicroscopy.org/artifactory/maven/, both the release as well as the SNAPSHOT versions should be available.

Comment (mtbc) [reply pom.xml]: > Could not find artifact ome:pom-omero-client:pom:5.5.0-SNAPSHOT in ome.maven (https://artifacts.openmicroscopy.org/artifactory/maven)

Comment (sbesson) [reply pom.xml]: Huh yes... this PR needs to be fixed and merged first (the deployment happens via Travis)

**Comment by jburel** [comment]
@sbesson updated

**Comment by sbesson** [comment]
I assume we'll need to review/test whether https://github.com/ome/pom-omero-client/pull/42/commits/a4299bf82a427ea1390b0eff3eb16d9e1ef35a4d breaks clients declaring `ice 36` as a property. I assume `omero-downloader` and `minimal-omero-client` will be the primary testers of this. Merging so that we have a SNAPSHOT we can start consuming/testing and when we are happy we can tag it as 5.5.0-m1

File: pom.xml (hunk 2 above)
Comment (sbesson) [reply pom.xml]: @mtbc now available at https://artifacts.openmicroscopy.org/artifactory/maven/ome/pom-omero-client/5.5.0-SNAPSHOT/

Comment (mtbc) [reply pom.xml]: @sbesson: Seems to be working fine with downloader pulling from eel, :+1:

Comment (mtbc) [reply pom.xml]:
```diff
--- a/pom.xml
+++ b/pom.xml
@@ -9,7 +9,7 @@
   <parent>
     <groupId>ome</groupId>
     <artifactId>pom-omero-client</artifactId>
-    <version>5.4.9</version>
+    <version>5.5.0-SNAPSHOT</version>
   </parent>
```

**Comment by mtbc** [comment]
Note: downloader seems to work despite defining `<ice.version>ice36</ice.version>` in the properties tag.

**Comment by sbesson** [comment]
I think the activation breakage would only happen if your pom.xml declared `<name>ice</name>` as a property. Good to know that declaring `<ice.version>ice36</ice.version>` directly keeps working.

**Pull request merged**